packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -155,11 +151,13 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
         _outerZones.add(this);
 
         if (windowElement && _outerZones.size === 1) {
-          _disposeGlobalKeyDownListener = on(windowElement, 'keydown', this._onKeyDownCapture, true);
+          windowElement.addEventListener('keydown', this._onKeyDownCapture, true);
```

Comment:
We need a better pattern for ensuring that all manually added listeners are also removed.

Reply:
An API and/or lint rule would be grand. We've cleaned-up a number of these from analyzing snapshots downstream, so it would be unfortunate to reintroduce them.

Reply:
Tracked in #11957